src/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/TripleDesImplementation.Windows.cs

```diff
@@ -2,8 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Security.Cryptography;
 using Internal.NativeCrypto;
+using System;
+using System.Security.Cryptography;
```

Comment:
I think System imports are supposed to come before internal imports. I liked it better the original way anyhow.